doc/changes/changes_2.1.2.md

```diff
@@ -0,0 +1,18 @@
+# Spark Connector 2.1.2, released 2023-??-??
+
+Code name:
```

Comment:
If you plan to create a release please add release date, code name & summary

Reply:
Not at the moment, we have couple of fixes in todo